app/src/main/res/values-it/strings.xml

```diff
@@ -171,216 +171,205 @@ Halten Sie sich an die Hygiene- und Verhaltensregeln."</string>
+    <string name="meldungen_meldung_title">Note</string>
+    <string name="meldungen_meldung_text">Una persona con cui hai avuto contatto è stata testata positiva al COVID-19. Leggi le informazioni e le raccomandazioni qui.</string>
+    <string name="meldungen_meldung_more_button">Più informazioni</string>
+    <string name="tracing_active_title">Tracciamento attivo</string>
+    <string name="tracing_active_text">Gli incontri sono registrati in forma anonima sul tuo device.</string>
+    <string name="tracing_error_title">Il tracciamento non è attivo.</string>
+    <string name="tracing_error_text">Attivare il Bluetooth, così da permettere di registrare gli incontri in forma anonima.</string>
 
     <!-- Meldungen: zusätzliche Info Titel -->
-    <string name="meldungen_additional_info_title">"Was bedeutet das?"</string>
+    <string name="meldungen_additional_info_title">Cosa vuol dire ?</string>
 
     <!-- Meldungen: Info Text 1 -->
-    <string name="meldungen_additional_info_text1">"Um die Verbreitung eines Virus zu stoppen, muss man die Ansteckungsketten unterbrechen. Das gelingt, wenn man ansteckende Personen isoliert. Nur: Beim neuen Coronavirus gibt es ein Problem. Es kann sein, dass du bereits ansteckend bist, bevor du dich krank fühlst. So kann sich das Virus ausbreiten, obwohl sich Personen mit Symptomen selbst isolieren. Denn wir sind einen Schritt zu spät."</string>
+    <string name="meldungen_additional_info_text1">Per bloccare il virus bisogna interrompere la catena di trasmissione. Questo si può fare isolando le persone contagiose. Con il COVID-19 le persone possono infette possono essere contagiose prima di mostrare i sintomi, il che sigifica che possono trasmettere il virus senza saperlo. Se solo le persone con sintomi vengono isolate il virus continuerà a trasmettersi. Dobbiame stare un passo avanti !</string>
 
     <!-- Meldungen: zusätzlicher Info Text -->
-    <string name="meldungen_additional_info_text2">"Deshalb sollen Kontakpersonen informiert werden. So können sie sich frühzeitig isolieren – noch bevor sie sich krank fühlen. Natürlich haben dann nicht alle Kontaktpersonen das Virus. Aber diejenigen, die es haben, geben es nicht weiter. So sind wir dem Virus einen Schritt voraus."</string>
 
     <!-- Hinweis beim Eingabefeld für den Authentifizierungs-Code -->
     <string name="auth_code_input_hint">"000-000"</string>
 
     <!-- Meldungen: Info Titel -->
-    <string name="meldungen_additional_info_title2">"Wie soll ich mich verhalten?"</string>
+    <string name="meldungen_additional_info_title2">Cosa devo fare ?</string>
 
     <!-- Meldungen: Text -->
-    <string name="meldungen_additional_info_text3">"Experten vom BAG beraten Sie telefonisch.
-
-Corona-Infoline
-+41 58 463 00 00"</string>
+    <string name="meldungen_additional_info_text3">Gli esperti di XXX ti daranno informazioni per telefono\\\nCoronavirus infoline +39 00 000 0000</string>
```

Comment:
I think the number to call in case of symptoms is the 112. [source](https://dilei.it/salute/coronavirus-cosa-fare-e-come-riconoscere-i-sintomi/689072/)

Reply:
Purtroppo è molto peggio, c'è un numero nazionale informativo e un numero diverso per ogni regione, più il 112 in caso di emergenza.
Vedremo come gestire questo.

Reply:
Mmh that's annoying.. Is Italy the only one that has a number for each region, or also other countries as far as you know?